document/docs/charts/pie-chart.md

```
@@ -48,4 +48,151 @@ PieChart(
+    PieChart(
+        ...,
+        startDegree = -90f
+    )
+```
+
+### Labeling
+
+Display labels directly on pie slices using the **OnPie** labeling mode.
+
+<div>
+   <img src="https://github.com/ehsannarmani/ComposeCharts/blob/master/assets/charts/pie_chart_label2.gif?raw=true" width="300">
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)